From ac3933776a59d75b26b525bb6fd2acd80659d865 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Tue, 15 Nov 2022 11:13:11 +0100
Subject: [PATCH] ARM: dts: stm32: add uart4 power domain on stm32mp151

Add power domain description for STM32 UART4 wake-up on STM32MP151.
UART4 is able to wake-up from LP-Stop mode, so it uses pd_core domain.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: I23531a3284674ca7f977df34fc76c2d8a5ed70c0
---
 arch/arm/boot/dts/st/stm32mp151.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/st/stm32mp151.dtsi b/arch/arm/boot/dts/st/stm32mp151.dtsi
index b73f99dab264..379fffa08522 100644
--- a/arch/arm/boot/dts/st/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp151.dtsi
@@ -519,6 +519,7 @@ uart4: serial@40010000 {
 			interrupts-extended = <&exti 30 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&rcc UART4_K>;
 			wakeup-source;
+			power-domains = <&pd_core>;
 			dmas = <&dmamux1 63 0x400 0x15>,
 			       <&dmamux1 64 0x400 0x11>;
 			dma-names = "rx", "tx";
-- 
2.39.5

